### PR TITLE
fix(deps): bump openTelemetryVersion from 1.19.0 to 1.27.0

### PR DESCRIPTION
**Pull request by JoergSiebahn**
see PLP-819

### CODE REVIEW

File: sda-commons-server-opentelemetry/src/main/java/org/sdase/commons/server/opentelemetry/OpenTelemetryBundle.java [hunk 1]
```diff
@@ -86,13 +87,16 @@ private static OpenTelemetry bootstrapConfiguredTelemetrySdk() {
...
           .build()
           .getOpenTelemetrySdk();
-    } catch (IllegalStateException e) {
-      LOG.warn(
-          "A global tracer already defined. "
-              + "This should only happen in a test that uses the OpenTelemetryExtension. "
-              + "If you see this in other circumstances, something with the setup is wrong.",
-          e);
-      return GlobalOpenTelemetry.get();
+    } catch (IllegalStateException | ConfigurationException e) {
```
Comment (sliesche) [inline sda-commons-server-opentelemetry/src/main/java/org/sdase/commons/server/opentelemetry/OpenTelemetryBundle.java:90]: not sure why this change is needed

**Review by sliesche — APPROVED** [review]

File: sda-commons-server-opentelemetry/src/main/java/org/sdase/commons/server/opentelemetry/OpenTelemetryBundle.java (hunk 1 above)
Comment (JoergSiebahn) [reply sda-commons-server-opentelemetry/src/main/java/org/sdase/commons/server/opentelemetry/OpenTelemetryBundle.java:90]: Most likely, [this](https://github.com/open-telemetry/opentelemetry-java/commit/6edba790d87782dd2d681ac90beaa51b74e95c78#diff-55d1160e6686c4da083f6744eb9a7d3a17c5f1ad4bafc36e54bfc65f9b42ac6eR404-R421) changed the exception type we need to catch. But I'm not 100% sure, if the `IllegalStateException` can't happen any more now.
